bazel/external_crates.bzl

```diff
@@ -1096,6 +1096,13 @@ def external_crates_repository(name, cargo_lockfile, lockfile, sanitizers_enable
             "scraper": crate.spec(
                 version = "^0.17.1",
             ),
+            "secp256k1": crate.spec(
```

Comment:
Do I understand correctly that we pull this in because we want to test with a library that we don't use in production?

Reply:
It's actually even more unfortunate situation

- The monorepo has two different `bitcoin` versions available, 0.28 and 0.32
- This PR was originally written against `bitcoin 0.32`
- However the way `bitcoin 0.32` is built in the monorepo, it's not possible to use it *except* in canisters. (See #1150). I couldn't figure out a way around this.
- OK so I'll rewrite the test util to use `bitcoin 0.28` API
- That works, *except* ...
- That version of `bitcoin` depends on a version of `secp256k1` where, if you want to verify any signatures, you must enable a special feature flag in `secp256k1`.
- Ideally the `bitcoin` crate itself would have a Rust feature, which if set enabled the relevant feature in the dependency.
- It does not.
- So we name this dependency in order to enable the feature such that we can verify the signatures

:sigh:

Reply:
:sigh: indeed! Thanks for the detailed explanation! That we can use bitcoin 0.32 only in WASM is quite unfortunate.